### PR TITLE
add Simplified Chinese

### PR DESCRIPTION
**Pull request by Hunlongyu**

### CODE REVIEW

**Comment by kazup01** [comment]
Thank you! I'll check it!

File: readme-zh-cn.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+</h1>
+<h4 align="center">程序员笔记应用程序 </h4>
+<h5 align="center">macOS, Windows and Linux</h5>
+<h5 align="center">由 Electron, React + Redux, Webpack and CSSModules 构建</h5>
+
+![Boostnote app screenshot](./resources/repository/top.png)
+
+[![Build Status](https://travis-ci.org/BoostIO/Boostnote.svg?branch=master)](https://travis-ci.org/BoostIO/Boostnote)
+
+## 作者 & 维护
```
Comment (asmsuechan) [inline readme-zh-cn.md]: Is this 维护者?

Comment (Hunlongyu) [inline readme-zh-cn.md]: Yeah, I didn't notice

**Comment by asmsuechan** [comment]
@Hunlongyu Thanks for your translation! But sorry, could you remove except readme-zh-cn.md?

**Comment by asmsuechan** [comment]
Hi, @Hunlongyu. What's going on?

**Comment by Hunlongyu** [comment]
I'm sorry, I'm busy recently @asmsuechan

**Comment by asmsuechan** [comment]
@Hunlongyu No problem! Thank you for replying :smile:

**Pull request closed without merging**